packages/ckeditor5-editor-classic/src/classiceditorui.ts

```diff
@@ -124,7 +118,11 @@ export default class ClassicEditorUI extends EditorUI {
 
 		this._initPlaceholder();
 		this._initToolbar();
-		this._initMenuBar();
+
+		if ( view.menuBarView ) {
+			_initMenuBar( this as EditorUI & { view: EditorUIView & { menuBarView: MenuBarView } } );
```

Comment:
Does this typing have to be this weird :grimacing:? Why these things are missing in `ClassicEditor`, can we do anything about it? It's ugly :D.

Reply:
`ClassicEditor` has `menuBarView?: MenuBarView` instead of `menuBarView: MenuBarView`, but I will try to improve this.